core/src/test/java/org/apache/calcite/test/MaterializationTest.java

```diff
@@ -2468,12 +2468,25 @@ private void checkSatisfiable(RexNode e, String s) {
     checkMaterialize(sql, sql);
   }
 
-  @Test public void testUnionToUnion() {
+  @Test public void testUnionToUnion0() {
     String sql0 = "select * from \"emps\" where \"empid\" < 300";
```

Comment:
Can we avoid to use the test case name XXX 0 .. 1.. 2, and give them more meaningful names ? Thus, we can see the test content difference just from the name.

Reply:
Thanks, updated ~

Reply:
Thanks, i'm fine with the latest change.

Reply:
Thanks a lot ~ Danny

Reply:
👍